packages/styles/scss/components/_socialmedia.scss

```diff
@@ -7,9 +7,10 @@
   $c: &;
   $default-theme: "light";
   $icon-size: $spacing-spacing-4;
+  $ilo-color-blue: "rgba(30, 45, 190, 1)";
```

Comment:
I think what the guys meant is to assign the CSS variable to the SCSS variable so that it can be passed into the mixin.

Reply:
You mean  $ilo-color-blue: var(--ilo-color-blue); right ? I tried this but it didnt seem to work.